src/Nancy.Tests/Unit/Diagnostics/DefaultRequestTraceFactoryFixture.cs

```diff
@@ -62,10 +62,7 @@ public void Should_initialize_case_insensitive_items_collection_when_case_sensit
                 trace.Items.Add("foo", "bar");
 
                 // Then
-                Assert.DoesNotThrow(() =>
-                {
-                    var value = trace.Items["FOO"];
-                });
+                var value = trace.Items["FOO"];

```

Comment:
Isn't the point of using Record to then assert the type of the expected
exception?

On Sunday, 1 November 2015, Damian Hickey <notifications@github.com> wrote:

> In src/Nancy.Tests/Unit/Diagnostics/DefaultRequestTraceFactoryFixture.cs
> <https://github.com/NancyFx/Nancy/pull/2098#discussion_r43585456>:
>
> > @@ -62,10 +62,7 @@ public void Should_initialize_case_insensitive_items_collection_when_case_sensit
> >                  trace.Items.Add("foo", "bar");
> >
> >                  // Then
> > -                Assert.DoesNotThrow(() =>
> > -                {
> > -                    var value = trace.Items["FOO"];
> > -                });
> > +                var value = trace.Items["FOO"];
>
> Record.Exception(() => trace.Items["FOO"])
>     .ShouldBeNull();
>
> ^ Look reasonable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2098/files#r43585456>.
>